tony-core/src/main/java/com/linkedin/tony/TonyJobMetadata.java

```diff
@@ -5,31 +5,28 @@
 package com.linkedin.tony;
 
 import com.linkedin.tony.util.Utils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
 
 
 public class TonyJobMetadata {
-  private static final Log LOG = LogFactory.getLog(TonyJobMetadata.class);
   private String id;
   private String url;
-  private long started;
-  private long completed;
+  private Long started;
+  private Long completed;
   private String status;
   private String user;
 
   // for testing only
   TonyJobMetadata() {
     this.id = "";
     this.url = "";
-    this.started = 0;
-    this.completed = 0;
+    this.started = (long) 0;
+    this.completed = (long) 0;
```

Comment:
Rather than casting an int to a long, you can create a long literal by using `0L`.

Reply:
👍